app/vlinsert/main.go

```diff
@@ -29,15 +36,27 @@ func Stop() {
 func RequestHandler(w http.ResponseWriter, r *http.Request) bool {
 	path := r.URL.Path
 
-	if path == "/internal/insert" {
+	switch {
+	case strings.HasPrefix(path, "/internal/insert/") || path == "/internal/insert":
```

Comment:
The `strings.HasPrefix(path, "/internal/insert/")` check isn't needed. It is enough to leave the `path == "/internal/insert"` check.